tests/New-DbaDbUser.Tests.ps1

```diff
@@ -12,7 +12,7 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
         }
     }
     Context "Should error out if the database does not exist" {
-        Mock -CommandName Connect-SqlInstance {
+        Mock -CommandName Connect-DbaInstance {
```

Comment:
Please revert this back, the `Connect-SqlInstance` is the function used in `New-DbaDbUser`.